planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/util/YAMLmatchmakerToOptimizerParser.java

```diff
@@ -39,18 +40,18 @@
 
    // Reducing verbosity . If somebody knows a better approach for doing this (I
    // could not set dynamically the level of the logging) it should be changed
-   private static int           BeeingTooVerboseWithLackOfInformationInCloudOffers = 3;
+   private static int BeeingTooVerboseWithLackOfInformationInCloudOffers = 3;
 
    static Logger log = LoggerFactory.getLogger(YAMLmatchmakerToOptimizerParser.class);
 
    @SuppressWarnings("unchecked")
    public static List<Object> getListofOptions(String appModel) {
       Yaml yamlApp = new Yaml();
-      if(log.isDebugEnabled()){
-      log.debug("Loading String to a YAML using sknakeyaml.Yaml");
+      if (log.isDebugEnabled()) {
+         log.debug("Loading String to a YAML using sknakeyaml.Yaml");
```

Comment:
Thank you @adriannieto  . The check improves slightly the performance because the string does not need to be created, the debug method invoked, and then ignored. It simply checks a condition and skips the rest of the process. 

OK, this string was not very complex to generate either...